package.json

```diff
@@ -3,8 +3,9 @@
   "version": "1.0.0",
   "private": true,
   "scripts": {
-    "start": "node server.js",
-    "dev": "cross-env NODE_ENV=development nodemon server.js",
+    "start": "node ./dist/server.js",
```

Comment:
Can we use ts-node instead of node?

Reply:
I don't think so we need to use `ts-node` as per my knowledge but if there are some extra benefits of using `ts-node` we can discuss this.